tests/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ endif()
 
 function(add_unit_test name)
   set(options DISABLED_TEST)
+  set(options NO_BSD_OVERLAY)
```

Comment:
This drops the `DISABLED_TEST` since you overwrite the variable.  Just append to it.

    set(options DISABLED_TEST;NO_BSD_OVERLAY)